packages/block-editor/src/components/global-styles/shadow-panel-components.js

```diff
@@ -60,11 +59,13 @@ export function ShadowPresets( { presets, activeShadow, onSelect } ) {
 	);
 }
 
-export function ShadowIndicator( { label, isActive, onSelect, shadow } ) {
+export function ShadowIndicator( { type, label, isActive, onSelect, shadow } ) {
 	return (
 		<div className="block-editor-global-styles__shadow-indicator-wrapper">
 			<Button
-				className="block-editor-global-styles__shadow-indicator"
+				className={ `block-editor-global-styles__shadow-indicator ${
+					type === 'unset' ? 'unset' : ''
+				}` }
```

Comment:
It would be better to use `classnames` library

Reply:
Updated the with `classnames` call